### PR TITLE
Add community tutorial subsection

### PR DESCRIPTION
**Pull request by rgreinho**
Adds a new subsection to collect the tutorials written by the Chains
community.

Fixes tektoncd/chains#329

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,30 +109,44 @@ for more information.
...
+To get started with Chains, try out our
+[getting started tutorial](docs/tutorials/getting-started-tutorial.md).
 
-To start signing OCI images and generating signed provenance for them, try our [signed provenance tutorial](docs/tutorials/signed-provenance-tutorial.md).
+To start signing OCI images and generating signed provenance for them, try our
+[signed provenance tutorial](docs/tutorials/signed-provenance-tutorial.md).
+
+### Community tutorials
+
+The Chains community has been hard at work is sharing some resources as well :
```
Comment (priyawadhwa) [inline README.md:126]:
```suggestion
The Chains community has been hard at work creating tutorials as well:
```

File: README.md [hunk 2]
```diff
@@ -98,30 +109,44 @@ for more information.
...
+
+- [Dual storage backend setup](https://github.com/thesecuresoftwarefactory/ssf/blob/main/examples/buildpacks/dual-storage-backend.md) showcases how to use multiple storage backends and
+verify the attestations with [cosign].
```
Comment (priyawadhwa) [inline README.md:129]: nit, missing the link to cosign

Comment (rgreinho) [reply README.md:129]: It is at the very bottom of the file.

Comment (priyawadhwa) [reply README.md:129]: i see it! but it still doesn't seem to be resolving 🤔 

image

Comment (rgreinho) [reply README.md:129]: Weird, I see it as resolved though:

image

**Review by priyawadhwa — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (priyawadhwa) [reply README.md:129]: let's merge and see 😄

**Pull request merged**